integrated-models/Fruitfly-GEM/compartmentSVG.tsv

```diff
@@ -0,0 +1,2 @@
+# compartment name must exist in the model
+# filename must be a .svg with only [a-zA-Z0-9_] characters
```

Comment:
I feel like the sentence reads better with `filename must has an extension svg and with ...`